internal/website/content/howto/server/_index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Server"
+date: 2019-06-21T10:36:43-07:00
+draft: true
+showInSidenav: true
+---
+
+The Go CDK's `server` package provides a pre-configured HTTP server with diagnostic hooks for request logging, health checks, and trace exporting via OpenCensus. These guides will show you how to start up and shut down the server, as well as how to work with the request logging, health checks, and trace exporting.
+
+The Go CDK includes a server package because??
+  from RBG: not only to provide some reasonable defaults (timeouts, logger), but also to show people what they *could* do eg healthchecks.
+
+## Starting up the server
+
+The GO CDK Server constructor takes an `http.Handler` and an `Options` struct. The simplest way to start the server is to use the `http.DefaultServeMux` and pass `nil` for the options.
+
+{{< goexample src="gocloud.dev/server.ExampleServer_New" >}}
+
+### Adding a request logger
+
+You can use the `server.Options` struct to specify a request logger.
+
+The example is shown with the Go CDK [`requestlog`](https://godoc.org/gocloud.dev/requestlog) package's `NCSALogger`. To get logs in the Stackdriver JSON format, use `NewStackdriverLogger` in place of `NewNCSALogger`.
+
+{{< goexample src="gocloud.dev/server.ExampleServer_RequestLogger" >}}
+
+### Adding health checks
+
+The Go CDK `server` package affords a hook for you to define health checks for your application and see the results at `/healthz/readiness`. Health checks are an imortant part of application monitoring.
+
+
+Because each application may have a different definition of what it means to be "healthy", you will need to define a concrete type to implment the `health.Checker` interface and define a `CheckHealth` method specific to your application.
+```go
```

Comment:
This code I had to copy because the example importer at the moment seems to only does actual example tests. (I tried adding the magic comment to both the struct and the function and importing them and using them like the example test and it did not work. Turns out you also cannot define a struct type and give it methods from within another function.)

Reply:
SG, I saw the docstore howto does the same